examples/rag/test_data/dummy_seq2seq/test.source

```diff
@@ -0,0 +1,2 @@
+What is love ?
```

Comment:
Do you want to leave those in master?

Reply:
If it's just for testing I'd rather create them at test time

Reply:
What do you recommend as an alternative ?
I can generate the dummy data on the fly in the test script for example